lib/utils.js

```diff
@@ -77,17 +77,25 @@ async function getAndCheckIosSdkVersion () {
 
 async function translateDeviceName (pv, dn = '') {
   let deviceName = dn;
-  if (dn.toLowerCase() === 'iphone simulator') {
+  dn = dn.toLowerCase().trim();
+  if (dn === 'iphone simulator') {
```

Comment:
can we use `switch` instead?

Reply:
IMHO it might make the stuff more readable, however its more a matter of taste (that is why Guido excluded `switch` from Python completely %) )